service/build.gradle

```diff
@@ -24,6 +25,14 @@ dependencyManagement {
     }
 }
 
+configurations {
+    // https://stackoverflow.com/questions/76706612/standard-commons-logging-discovery-in-action-with-spring-jcl
+    // resolves "Standard Commons Logging discovery in action with spring-jcl: please remove commons-logging.jar from classpath in order to avoid potential conflicts"
+    configureEach {
+        exclude group: 'commons-logging', module: 'commons-logging'
```

Comment:
would it be worth removing this specifically from each dependency instead of doing it for all our dependencies?

Reply:
probably yes... i was being lazy because when i took it out of the dependency i expected, it didn't get rid of the warning. i'll work on this

Reply:
offline conversation: leave as-is for now